doc/DeveloperGuide.md

```diff
@@ -94,6 +94,7 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should be intuitive to users
```

Comment:
this is definitely wanted by all users! Perhaps you can be a little more concrete about what is intuitive? If this guide were to be presented to you how would you interpret your own requirements?